website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -178,11 +178,21 @@ You can follow the logs with the following command:
 docker logs -f appsmith
 ```
 
-This message is also logged and available in server logs in (`stacks/logs/backend/backend.log`).
+This message is also logged and available in server logs in (`stacks/logs/backend/backend.log`). You can access backend logs from your Appsmith deployment by following these steps:
+
```

Comment:
I think the below steps can be shown in a video. cc: @jnikhila what do you think?

Reply:
in fact we can just link to felix's document - https://github.com/felix-appsmith/guide-how-to-get-logs-appsmith

Reply:
I think the ticket mentioned that we need an official doc in docs.appsmith

Reply:
i agree but in our docs we don't need to put a context around how the log files can be shared with the support team. So for eg the line "Retrieve the .log files and bundle them in a .zip file." sounds a bit  odd as the reader does not understand the context. We should definitely mention the backend logs which was already there, but for some reason it was not discoverable because of that single line statement right under front end logs.